code/modules/surgery/organs/organ.dm

```diff
@@ -248,13 +248,18 @@ var/list/organ_cache = list()
 		return
 	switch (severity)
 		if (1.0)
-			take_damage(0,20)
+			take_damage(20,0)
```

Comment:
This is not good, this will make robotic organs take brute damage instead of burn damage

You'll want to override emp_act for internal organs, and have it do this instead, with `silent` set to 1 due to a lack of nerves